### PR TITLE
[Uxd 570] add bootcamp page

### PR DESCRIPTION
**Pull request by ThyeeZz**

### CODE REVIEW

File: src/components/selector/v2/index.jsx [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
   options,
-  setSelected = () => { },
   className = '',
+  locale
 }) => {
   const choosenWrapper = useRef(null);
   const [open, setOpen] = useState(false);
-
+  const label = locale === 'cn' ? '中文' : 'English';
+  console.log(label);
```
Comment (nameczz) [inline src/components/selector/v2/index.jsx:15]: delete console

File: src/pages/bootcamp/index.module.less [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    .title {
+      font-family: 'inter Bold';
+      font-size: 24px;
+      line-height: 32px;
+      margin: 64px 0 20px;
+
+      &:lang(cn) {
+        font-family: '思源黑体';
```
Comment (nameczz) [inline src/pages/bootcamp/index.module.less:33]: check font family

File: src/pages/bootcamp/index.jsx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+      <main className={styles.mainContainer}>
+        <BannerCard
+          title={title}
+          content={description}
+          img={banner}
+          isMobile={isMobile}
+          locale={locale}
+        />
+        <div className={styles.benchmarkContainer}>
+          <p className={styles.title} lang={locale}>{benchmarkTitle}</p>
```
Comment (shanghaikid) [inline src/pages/bootcamp/index.jsx:63]: Title should be H1

File: src/pages/bootcamp/index.jsx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                  />
+                );
+              })
+            }
+          </div>
+
+        </div>
+
+        <div>
+          <p className={styles.title} lang={locale} >{solutionTitle}</p>
```
Comment (shanghaikid) [inline src/pages/bootcamp/index.jsx:84]: solution should be list, please use ul,li

**Review by shanghaikid — CHANGES_REQUESTED** [review]

File: src/components/header/v2/secondHeader.jsx [hunk 5]
```diff
@@ -108,17 +106,18 @@ const SecondHeader = ({
             tabIndex={-1}
           >
             {tabList.map(item => {
-              const { id, label } = item;
+              let { id, label, href } = item;
+              href = locale === 'en' ? href : `/cn${href}`;
```
Comment (Tumao727) [inline src/components/header/v2/secondHeader.jsx:110]: don't directly change parameters,  use another variable or put the statement in <Link />

File: src/components/card/LinkCard/index.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+      target='_blank'
+      rel="noreferrer"
+    >
+      <span className={styles.label}>{label}</span>
+      <img src={LinkIcon} alt="link-icon" className={styles.icon} />
+    </a>
+  );
+};
+
+export default LinkCard;
```
Comment (shanghaikid) [inline src/components/card/LinkCard/index.jsx:23]: no new line at the end of file

File: src/components/card/LinkCard/index.module.less [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  height: 100%;
+
+  .label {
+    display: inline-block;
+    font-family: 'Inter Bold';
+    font-size: 20px;
+    line-height: 28px;
+
+    &:lang(zh-cn) {
+      font-family: '思源黑体';
```
Comment (shanghaikid) [inline src/components/card/LinkCard/index.module.less:23]: same problem

File: src/components/card/bannerCard/index.jsx [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+              <img src={img} alt="banner" className={styles.img} />
+            </div>
+          </div>
+        )
+      }
+    </>
+  );
+};
+
+export default BannerCard;
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.jsx:58]: no new line at the end of file

File: src/components/card/bannerCard/index.jsx [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+  className = '',
+  img,
+  isMobile = false
+}) => {
+
+  return (
+    <>
+      {
+        isMobile ? (
+          <div className={`${styles.mobileBannerCard} ${className}`}>
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.jsx:17]: can we just use css to do this mobile styling? flex order?

File: src/components/card/bannerCard/index.module.less [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+    .text {
+      width: 100%;
+      white-space: normal;
+      word-break: break-word;
+      font-family: 'Inter';
+      font-size: 16px;
+      line-height: 26px;
+
+      &:lang(zh-cn) {
+        font-family: '思源黑体';
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.module.less:25]: same font issue

File: src/components/card/bannerCard/index.module.less [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+    width: 33.33%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .text {
+      width: 100%;
+      white-space: normal;
+      word-break: break-word;
+      font-family: 'Inter';
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.module.less:20]: what is the fallback of this font?

File: src/components/card/bannerCard/index.module.less [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+        }
+      }
+
+      & + p {
+        margin-top: 16px;
+      }
+    }
+
+    .title {
+      font-family: 'Inter Bold';
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.module.less:40]: fallback please

File: src/components/card/bannerCard/index.module.less [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+      font-size: 24px;
+      line-height: 32px;
+      margin-bottom: 16px;
+
+      &:lang(zh-cn) {
+        font-family: '思源黑体';
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.module.less:46]: same font

File: src/components/card/bannerCard/index.module.less [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+  .text {
+    width: 100%;
+    white-space: normal;
+    word-break: break-word;
+    font-family: 'Inter';
+    font-size: 16px;
+    line-height: 26px;
+
+    &:lang(zh-cn) {
+      font-family: '思源黑体';
```
Comment (shanghaikid) [inline src/components/card/bannerCard/index.module.less:81]: why there are so many zh-cn font settings ? can we just use one ?

File: src/components/card/solutionCard/index.module.less [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+  .content {
+    font-family: 'Inter';
+    font-size: 15px;
+    line-height: 24px;
+    width: 100%;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 2; //行数
```
Comment (shanghaikid) [inline src/components/card/solutionCard/index.module.less:48]: remove chinese

File: src/pages/index.less [hunk 16]
```diff
@@ -9,11 +9,10 @@ body {
   width: 100%;
   background-color: #fff;
 
-  .content {
-    max-width: @screen-lg-max;
+  .page-content {
+    max-width: 1400px;
```
Comment (shanghaikid) [inline src/pages/index.less:13]: why 1400, what is the value of @screen-lg-max

**Review by shanghaikid — CHANGES_REQUESTED** [review]

**Pull request merged**